test/examples/bootstrap.js

```diff
@@ -5,6 +5,15 @@ const puppeteer = require('puppeteer');
 const net = require('net');
 const fs = require('fs');
 const http = require('http');
+// this line allows to disable a warning generated by node when more than 10
```

Comment:
10 or 100 ?

Reply:
The default value set by node is 10, I set it to 100 to avoid the warning. I can precise the comment if you think its necessary.

Reply:
Óh OK I understand, don't worry no problem !